pandas/core/groupby.py

```diff
@@ -929,6 +929,10 @@ def mean(self):
         Compute mean of groups, excluding missing values
 
         For multiple groupings, the result index will be a MultiIndex
+
+        Do not call this function via the numpy library. numpy's mean
```

Comment:
Okay, so I understand putting off groupby/window function changes for a separate PR.  I'm a little confused though about not wanting users to use `numpy`.  So I shouldn't put any note in the documentation about not calling these functions/methods from the `numpy` library and then we're adding further compat to make it even more okay to call them in that manner?